doc/development/deprecations.rst

```diff
@@ -6,6 +6,11 @@ Deprecations
 Pending deprecations
 --------------------
 
+* ``LieAlgebraOptimizer`` is deprecated. Please use ``RiemannianGradientOptimizer`` instead.
```

Comment:
```suggestion
* ``LieAlgebraOptimizer`` is renamed. Please use ``RiemannianGradientOptimizer`` instead.
```

Might be better to just say "renamed" instead of "deprecated". No behaviour is actually changing.

Reply:
Makes sense!